src/commands/new.ts

```diff
@@ -364,319 +367,335 @@ export default {
       hr()
       p2()
-      p2("To run in Android, make sure you've followed the latest")
-      p2(`react-native setup instructions. You reference them at:`)
-      p2(`${link("https://reactnative.dev/docs/environment-setup")}`)
+      p2("Need additional help?")
+      p2()
+      p2(`Join our Slack community at ${link("http://community.infinite.red.")}`)
       p2()
-    }
-
-    hr()
-    p2()
-    p2("Need additional help?")
-    p2()
-    p2(`Join our Slack community at ${link("http://community.infinite.red.")}`)
-    p2()
-
-    hr()
-    p2()
-    p2("Now get cooking! 🍽")
-    command(`cd ${targetPath}`)
-    if (!installDeps) command(packager.installCmd({ packagerName }))
-    command(`${packagerName} start`)
-
-    const isMac = process.platform === "darwin"
-    if (isMac) {
-      command(`${packager.runCmd("ios", packagerOptions)}`)
-    } else {
-      command(`${packager.runCmd("android", packagerOptions)}`)
-    }
 
-    p2()
-    p2("With Expo:")
-    command(`cd ${projectName}`)
-    if (!installDeps) command(packager.installCmd({ packagerName }))
-    command(`${packager.runCmd("expo:start", packagerOptions)}`)
-    p2()
-    p2()
-    // #endregion
+      hr()
+      p2()
+      p2("Now get cooking! 🍽")
+      command(`cd ${targetPath}`)
+      if (!installDeps) command(packager.installCmd({ packagerName }))
+      command(`${packagerName} start`)
+
+      const isMac = process.platform === "darwin"
+      if (isMac) {
+        command(`${packager.runCmd("ios", packagerOptions)}`)
+      } else {
+        command(`${packager.runCmd("android", packagerOptions)}`)
+      }
 
-    // this is a hack to prevent the process from hanging
-    // if there are any tasks left in the event loop
-    // like I/O operations to process.stdout and process.stderr
-    // see https://github.com/infinitered/ignite/issues/2084
-    process.exit(0)
+      p2()
+      p2("With Expo:")
+      command(`cd ${projectName}`)
+      if (!installDeps) command(packager.installCmd({ packagerName }))
+      command(`${packager.runCmd("expo:start", packagerOptions)}`)
+      p2()
+      p2()
+      // #endregion
+
+      // this is a hack to prevent the process from hanging
+      // if there are any tasks left in the event loop
+      // like I/O operations to process.stdout and process.stderr
+      // see https://github.com/infinitered/ignite/issues/2084
+      process.exit(0)
+    } catch (e) {
+      stopLastSpinner("❌")
+      p2(red(`\nThe following error occured:\n${e}`))
+      startSpinner(" Gathering system and project details")
+      try {
+        const IGNITE = "node " + filesystem.path(__dirname, "..", "..", "bin", "ignite")
+        const doctorResults = await system.run(`${IGNITE} doctor`)
+        p(`\n\n${doctorResults}`)
+      } catch (e) {
+        p(yellow("Unable to gather system and project details."))
+      }
+      stopSpinner(" Gathering system and project details", "🛠️")
+      process.exit(0)
```

Comment:
This should be `process.exit(1)`, since it was unsuccessful. (`0` is successful, nonzero is failure)